### DISCUSSION TITLE
Env always return null after update packages to bref 2 version

### DISCUSSION THREAD

**Discussion by Rodevillow** [brefphp/bref#1805] [Support]
My project was on laravel 9 and I implement MaintenanceMode, and I notice the problem my env localy works grate, but for stage on aws after I update this packages my env always return null. I updated laravel to latest 11 version, but it's not resolved this problem.

That is my current configuration for bref packages in my composer.json

<img width="500" alt="Снимок экрана 2024-05-10 в 13 32 10" src="https://github.com/brefphp/bref/assets/13442582/2ce207ec-f19c-4f10-8645-553d6e57caac">

**Comment by mnapoli** [comment]
I'm moving this to discussions. Bref v2 was released a year ago, this is the first time I hear of this. Unlikely to be caused by the upgrade. I'd suggest to keep investigating, something wrong must be on your side TBH.
